Pythonwin/pywin/framework/startup.py

```diff
@@ -20,8 +20,8 @@
 	# sys.argv and it will be done here. ( This was a workaround for a bug in
 	# win32ui but is retained for other situations. )
 	argv = win32api.CommandLineToArgv(win32api.GetCommandLine())
-	sys.argv = argv[1:]
-	if os.getcwd() not in sys.path and '.' not in sys.path:
+	sys.argv = argv[1:] or ['']
```

Comment:
why this change?

Reply:
* according Python standard the init'ed sys.argv shall not be a empty list but [''].
* CWD can come in as '' and should be 1st in sys.path (unless a certain script is run in normal python)